Sources/App/Migrations.swift

```diff
@@ -41,9 +49,13 @@ class Migrations {
             
             app.databases.use(.postgres(configuration: postgresConfig), as: .psql)
             
-            app.autoMigrate()
         } catch {
             app.logger.error("Failed to connect to DB with error \(error)")
         }
+        do {
+            try app.autoMigrate().wait()
```

Comment:
I realised it wasn't actually waiting? it was just calling wait and then getting on with its business 🤔